.github/workflows/dotnetcore2.1.yml

```diff
@@ -0,0 +1,32 @@
+name: .NET Core
+
+on: [ push, pull_request ]
```

Comment:
```suggestion
on: [ push ]
```

Reply:
I disabled the 2.1 flow. Was not able to get it working so I'll create a ticket to look into it (if we even want it as it's been unsupported for a while now)